acceptance/cmd/workspace/apps/run-local/script

```diff
@@ -0,0 +1,31 @@
+cd app
+
+# We first run the command with different entry point which starts unblocking script
+# so we don't need to start it in background. It will install the dependencies as part of the command
+trace $CLI apps run-local --prepare-environment --entry-point test.yml 2>&1 | grep -w "Hello, world"
+
+title "Starting the app in background..."
+$CLI apps run-local --prepare-environment --debug > ../out.run.txt 2>&1 &
+PID=$!
```

Comment:
You can `trap` the exit to kill this process in case anything goes wrong.

It could zombie otherwise.